web/src/message_view.js

```diff
@@ -1031,6 +1031,9 @@ export function narrow_by_recipient(target_id, opts) {
     opts = {then_select_id: target_id, ...opts};
     // don't use message_lists.current as it won't work for muted messages or for out-of-narrow links
     const message = message_store.get(target_id);
+    const emails = message.reply_to.split(",");
+    const recipient_user_ids = emails.map((email) => people.get_user_id(email));
```

Comment:
Can we do this step of getting `user_id`s within the `sort_emails` function? We're repeating this in two places and also assuming the emails array is in same order as `user_id`s array within that function. That could be true with your usage of that function now, but might go wrong later.